spec/models/metrics/rejection_percentage_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
+
+RSpec.describe Metrics::RejectionPercentage do
+  Rejection::REASONS.each do |reason|
+    it "##{reason} defaults to 0" do
+      expect(subject.public_send(reason)).to eq(0)
+    end
+  end
+
+  describe '#attributes' do
```

Comment:
What I'm thinking is that this model is for the view that calculates the rejection percentages, so would be good to cover the case of when there are rejections to ensure that the values are correct.